### PR TITLE
Also add ATen/core tests to oss CI

### PR DESCRIPTION
**Pull request by smessmer**
Summary: -

Differential Revision: D9070030

### CODE REVIEW

File: aten/CMakeLists.txt [hunk 1]
```diff
@@ -135,6 +135,9 @@ list(APPEND ATen_CPU_INCLUDE
   ${CMAKE_CURRENT_BINARY_DIR}/src/ATen)
 add_subdirectory(src/ATen)
 
+file(GLOB tmp ${CMAKE_CURRENT_SOURCE_DIR}/src/ATen/test/core/*.cpp)
+set(Caffe2_CPU_TEST_SRCS ${Caffe2_CPU_TEST_SRCS} ${tmp})
```
[the remark this answers is not in the document]
Comment (ezyang) [reply aten/CMakeLists.txt]: And then you wouldn't need two cmake files, you can just put it all in `ATen/core/CMakeLists.txt`

[the remark this answers is not in the document]
Comment (ezyang) [reply aten/CMakeLists.txt]: Also, why are you monkeying around with `Caffe2_CPU_TEST_SRCS` directly?

[the remark this answers is not in the document]
Comment (smessmer) [reply aten/CMakeLists.txt]: don't have a strong opinion, moved it to the same directory. Also not monkeying around anymore ;)

File: aten/CMakeLists.txt [hunk 2]
```diff
@@ -135,6 +135,9 @@ list(APPEND ATen_CPU_INCLUDE
   ${CMAKE_CURRENT_BINARY_DIR}/src/ATen)
 add_subdirectory(src/ATen)
 
+file(GLOB ATen_CORE_TEST_SRCS "*_test.cpp")
+EXCLUDE(ATen_CPU_SRCS "${ATen_CPU_SRCS}" ${ATen_CORE_TEST_SRCS})
```
Comment (ezyang) [inline aten/CMakeLists.txt]: This looks like copy-paste. Can't you propagate it from aten/src/ATen/CMakeLists.txt by having that cmake set it to PARENT_SCOPE?

Comment (smessmer) [reply aten/CMakeLists.txt]: fixed

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**